.github/workflows/cargo-fmt.yml

```diff
@@ -4,37 +4,10 @@ jobs:
   cargo-fmt:
     runs-on: ubuntu-latest
     steps:
-      - name: install prerequisites
-        run: |
-          # make sure we don't silently continue
-          set -euo pipefail
-          sudo apt-get update --quiet=2
-          sudo apt-get install --yes meson nasm gcc-multilib
-        env:
-          DEBIAN_FRONTEND: noninteractive
       - name: git checkout
         uses: actions/checkout@v3
         with:
           fetch-depth: 1
-      - name: cache rust toolchain
-        uses: actions/cache@v3
-        with:
-          path: |
-            ~/.rustup/toolchains
-            ~/.rustup/update-hashes
-            ~/.rustup/settings.toml
-          key: ${{ runner.os }}-rust-toolchain-${{ hashFiles('rust-toolchain', 'rust-toolchain.toml') }}
```

Comment:
I think we still need this part, right?

Reply:
Doesn't seem like it? The workflow run succeeded and ran quickly, so I don't think we need to worry about caching for the fmt workflow.

Reply:
I guess it doesn't need it, since it'll download them anyways, which doesn't take *that* long, especially compared to the workflows that actually build and test, but if we already have this caching step in other workflows, I think it'd be nice to keep parity between them.  It would speed up formatting (usually only takes a second, not 18-24).

Reply:
Okay fair enough, I restored the caching step.